canon/CHANGELOG.md

```diff
@@ -0,0 +1,84 @@
+# 0.6.0
+
+## Changed
+
+Deprecate the usage of the `Store` trait.
+
+the trait `Canon<S>` becomes `Canon`.
```

Comment:
As mentioned in #79 the item should always start with one of the changelog's verb (see #79 for further details, and [dusk-abi's changelog](https://github.com/dusk-network/dusk-abi/blob/master/CHANGELOG.md) as example.

This is applying to the whole file, I'm not going to repeat myself in the review.